### ISSUE TITLE
ScrollReveal needs a license key

### ISSUE THREAD

**Issue by chrysochos** [pvtl/voyager-frontend#53]
During the installations I had the following response, that needs "Love ScrollReveal? 🔑  Buy a license!" . 
In order to avoid the problem I commented the corresponding 6 lines of code in vendor/pvtl/voyager-frontend/src/Commands/InstallCommand.php, but I don't know yet, what I left out.

Publishing complete.
Updating Root package.json to include dependencies

   Symfony\Component\Process\Exception\ProcessFailedException  : The command "
            npm install jquery what-input foundation-sites scrollreveal motion-ui --save-dev &&
            npm run dev
        " failed.

Exit Code: 1(General error)

Working directory: /var/www/html/voyager

Output:
================

> scrollreveal@4.0.5 postinstall /var/www/html/voyager/node_modules/scrollreveal
> node -e "console.log('\u001b[35m\u001b[1mLove ScrollReveal? 🔑  Buy a license!\u001b[22m\u001b[35m\n >> \u001b[33mhttps://scrollrevealjs.org/pricing/\u001b[0m\n')"

Love ScrollReveal? 🔑  Buy a license!
 >> https://scrollrevealjs.org/pricing/

**Comment by mathamel** [comment]
Got the same error, waiting on solution.

**Comment by mcnamee** [comment]
@chrysochos @mathamel 
This looks to be an 'ad' from [ScrollReveal](https://github.com/scrollreveal/scrollreveal#license). My understanding is that you can use scroll reveal for open source projects and non-commercial use - however if you decide to use it for commercial purposes, you need to purchase a licence.

In any case, given that many of us may use `voyager-frontend` for commercial purposes, I'd recommend we move away from ScrollReveal. Any suggestions for an alternative?

Can someone submit a PR?

- Update the install script to an appropriate alternative: https://github.com/pvtl/voyager-frontend/blob/master/src/Commands/InstallCommand.php#L92
- Update any page blocks that uses the animate function: https://github.com/pvtl/voyager-page-blocks/tree/master/resources/views/blocks
    - eg. https://github.com/pvtl/voyager-page-blocks/blob/master/resources/views/blocks/callout.blade.php#L9

**Comment by DaveArm** [comment]
I'm also getting the same error - any idea how I can get around this? I'm building locally testing to see if this is something I want to deep dive into before I unleash on a live project.

```
Symfony\Component\Process\Exception\ProcessFailedException  : The command "
            npm install jquery what-input foundation-sites scrollreveal motion-ui --save-dev &&
            npm run dev
        " failed.

Exit Code: 1(General error)
```

**Comment by mcnamee** [comment]
I've taken a deeper look at this. There is an error on the NPM component of `voyager-frontend:install`, however it doesn't have anything to do with ScrollReveal. The installation actually finishes successfully, it just doesn't compile the static assets (so just requires running `npm run dev` after the installation).

If you read further down in the error output (after the scroll reveal promotion), it says:

```bash
...
Error Output:
================
sh: 1: cross-env: not found
...
```

So it's failing at the Laravel Mix command: `npm run dev` (run by `voyager-frontend:install` command). I suspect it's got something to do with cross-env not being accessible within the Laravel command.

Either way, I've removed `npm run dev` from the voyager-frontend:install command, given that our instructions ask you to run it manually after installing.

I've tested (v0.9.2) on a fresh install and it installs successfully.
